iis-wf/iis-wf-referenceextraction/src/main/resources/eu/dnetlib/iis/wf/referenceextraction/covid19/main/oozie_app/workflow.xml

```diff
@@ -0,0 +1,233 @@
+                --conf spark.eventLog.dir=${nameNode}${spark2EventLogDir}
+            </spark-opts>
+            <arg>-input = ${input_document_metadata}</arg>
+            <arg>-output = ${workingDir}/transformed_meta</arg>
+        </spark>
+        <ok to="main_sqlite"/>
+        <error to="fail"/>
+    </action>
+
+    <action name="main_sqlite">
+        <map-reduce>
+            <prepare>
+                <delete path="${nameNode}${output}"/>
+            </prepare>
+            <streaming>
+            	<!-- Here, we give the relative path to the script and pass it
+            	the parameters of the workflow node. The script is held
+            	in a directory having the same name as the workflow node.
+
+            	The parameters should be passed as **named** arguments. This
+            	convention of passing them as named arguments makes the code
+            	more readable/maintainable.
+            	 -->
+                <mapper>scripts/madis/mexec.py -f scripts/covid19extract.sql</mapper>
+            </streaming>
+            <configuration>
+            	<!-- # Standard settings for our framework -->
+                <property>
+                    <name>mapred.output.format.class</name>
+                    <value>com.cloudera.science.avro.streaming.AvroAsJSONOutputFormat</value>
+                </property>
+                <property>
+                    <name>mapred.input.format.class</name>
+                    <value>com.cloudera.science.avro.streaming.AvroAsJSONInputFormat</value>
+                </property>
+                <!-- # Custom settings for this workflow node -->
+                <!-- We do not use any reducers, so we set their number to 0 -->
+                <property>
+                    <name>mapreduce.job.reduces</name>
+                    <value>0</value>
+                </property>
+                
+                <!-- INPUT -->
+                <property>
+                    <name>mapreduce.input.fileinputformat.inputdir</name>
+                    <value>${workingDir}/transformed_meta</value>
+                </property>
+
+                <property>
+                    <name>input.schema.literal</name>
+                    <value>${wf:actionData('generate-schema')['eu.dnetlib.iis.referenceextraction.covid19.schemas.DocumentMetadata']}</value>
+                </property>
+
+				<!-- OUTPUT -->
+                <property>
+                    <name>mapreduce.output.fileoutputformat.outputdir</name>
+                    <value>${workingDir}/matched_documents</value>
+                </property>          
+
+                <property>
+                    <name>output.schema.literal</name>
+                    <value>${wf:actionData('generate-schema')['eu.dnetlib.iis.referenceextraction.covid19.schemas.MatchedDocument']}</value>
+                </property>
+
+				<!-- this one is requred due to the large amount of time taken 
```

Comment:
I am also reducing this the timeout value from `28800000` to `7200000` because covid-19 mining performance is far better from the one I have used as a template for the workflow definition.